core/src/com/unciv/ui/worldscreen/AlertPopup.kt

```diff
@@ -244,6 +245,46 @@ class AlertPopup(val worldScreen: WorldScreen, val popupAlert: PopupAlert): Popu
                 addGoodSizedLabel(civInfo.nation.startIntroPart2).row()
                 add(getCloseButton("Let's begin!"))
             }
+            AlertType.DiplomaticMarriage -> {
+                val city = worldScreen.gameInfo.getCities().first { it.id == popupAlert.value }
+                addGoodSizedLabel(city.name.tr() + ": " + "What would you like to do with the city?".tr(), 24) // Add name because there might be several cities
```

Comment:
Doesn't addGoodSizedLabel translate, so you could do with "[${city.name}]: [What]"? Or the {} way.

Reply:
It does, I just wanted to save on a string. What's the {} way?

Reply:
Read Translations  `fun String.tr()` line 263 `if (contains('{'))` or [wiki](https://github.com/yairm210/Unciv/wiki/Translating#rules-for-templates-added-manually) - and improve that if you can :wink: